<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>651</width>
    <height>492</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(172, 172, 172);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="btCadFuncionario">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>20</y>
      <width>271</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Dubai</family>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(165, 165, 165, 255), stop:1 rgba(255, 255, 255, 255));
border-radius: 5px;
border-style: outset;
border-width: 1px;
border-color: black;
}
QPushButton:hover{
background-color: qlineargradient(spread:reflect, x1:0, y1:0, x2:1, y2:0.943182, stop:0 rgba(255, 126, 248, 255), stop:1 rgba(119, 146, 220, 255));
}</string>
    </property>
    <property name="text">
     <string>Cadastrar Funcion√°rio</string>
    </property>
   </widget>
   <widget class="QLabel" name="logoLoCarPequeno">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>0</y>
      <width>71</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(172, 172, 172);</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>../../AppData/PycharmProjects/LoCar/imagens/808107201585380077-64.png</pixmap>
    </property>
   </widget>
   <widget class="QPushButton" name="btCadCliente">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>20</y>
      <width>231</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Dubai</family>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(165, 165, 165, 255), stop:1 rgba(255, 255, 255, 255));
border-radius: 5px;
border-style: outset;
border-width: 1px;
border-color: black;
}
QPushButton:hover{
background-color: qlineargradient(spread:reflect, x1:0, y1:0, x2:1, y2:0.943182, stop:0 rgba(255, 126, 248, 255), stop:1 rgba(119, 146, 220, 255));
}</string>
    </property>
    <property name="text">
     <string>Cadastrar Cliente</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btVoltar">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>410</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Dubai</family>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(255, 255, 255, 255), stop:1 rgba(165, 165, 165, 255));
border-radius: 5px;
border-style: outset;
border-width: 1px;
border-color: black;

}

QPushButton:hover{
background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(255, 248, 0, 255), stop:1 rgba(255, 134, 73, 255));
}

</string>
    </property>
    <property name="text">
     <string>Voltar</string>
    </property>
   </widget>
   <widget class="QFrame" name="frameCadFuncionario">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>90</y>
      <width>611</width>
      <height>301</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(227, 227, 227);
border-radius: 5px;
border-style: outset;
border-width: 1px;
border-color: black;</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLineEdit" name="tbNomeFuncionario">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>70</y>
       <width>291</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);
</string>
     </property>
     <property name="placeholderText">
      <string>Nome</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="tbLoginFuncionario">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>110</y>
       <width>113</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);</string>
     </property>
     <property name="placeholderText">
      <string>Login</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="tbSenhaFuncionario">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>150</y>
       <width>113</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);</string>
     </property>
     <property name="echoMode">
      <enum>QLineEdit::Password</enum>
     </property>
     <property name="placeholderText">
      <string>Senha</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="tbMatriculaFuncionario">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>190</y>
       <width>113</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="placeholderText">
      <string>Matricula</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btCadastrarFuncionario">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>240</y>
       <width>111</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{
background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(255, 255, 255, 255), stop:1 rgba(165, 165, 165, 255));
border-style: outset;
border-width: 1px;
border-color: black;
border-radius: 5px;
}

QPushButton:hover{
background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(19, 226, 4, 255), stop:1 rgba(148, 209, 6, 255));
}


</string>
     </property>
     <property name="text">
      <string>Cadastrar</string>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="frameCadCliente">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>90</y>
      <width>611</width>
      <height>301</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(227, 227, 227);
border-radius: 5px;
border-style: outset;
border-width: 1px;
border-color: black;</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLineEdit" name="tbNomeCliente">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>40</y>
       <width>291</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);
</string>
     </property>
     <property name="placeholderText">
      <string>Nome</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="tbLoginCliente">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>100</y>
       <width>113</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);</string>
     </property>
     <property name="placeholderText">
      <string>Login</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="tbSenhaCliente">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>130</y>
       <width>113</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);</string>
     </property>
     <property name="echoMode">
      <enum>QLineEdit::Password</enum>
     </property>
     <property name="placeholderText">
      <string>Senha</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="tbCPFCliente">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>160</y>
       <width>113</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="placeholderText">
      <string>CPF</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btCadastrarCliente">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>240</y>
       <width>111</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{
background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(255, 255, 255, 255), stop:1 rgba(165, 165, 165, 255));
border-style: outset;
border-width: 1px;
border-color: black;
border-radius: 5px;
}

QPushButton:hover{
background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(19, 226, 4, 255), stop:1 rgba(148, 209, 6, 255));
}


</string>
     </property>
     <property name="text">
      <string>Cadastrar</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="tbCNHCliente">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>190</y>
       <width>113</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="placeholderText">
      <string>CNH</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="tbNCartaoCliente">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>220</y>
       <width>113</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="placeholderText">
      <string>n¬∫ Cart√£o</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="tbTelefoneCliente">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>250</y>
       <width>111</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="placeholderText">
      <string>Telefone</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="tbEnderecoCliente">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>70</y>
       <width>291</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="placeholderText">
      <string>Endere√ßo</string>
     </property>
    </widget>
   </widget>
   <zorder>logoLoCarPequeno</zorder>
   <zorder>btCadFuncionario</zorder>
   <zorder>btCadCliente</zorder>
   <zorder>btVoltar</zorder>
   <zorder>frameCadCliente</zorder>
   <zorder>frameCadFuncionario</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>651</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
